wherehows-dao/src/main/java/wherehows/dao/DatasetColumnRowMapper.java

```diff
@@ -56,21 +56,22 @@ public DatasetColumn mapRow(ResultSet rs, int rowNum) throws SQLException {
         String strDistributed = rs.getString(DISTRIBUTED_COLUMN);
         boolean distributed = isNotBlank(strDistributed) && strDistributed.equalsIgnoreCase("y");
         DatasetColumn datasetColumn = new DatasetColumn();
-        datasetColumn.id = id;
-        datasetColumn.sortID = sortID;
-        datasetColumn.parentSortID = parentSortID;
-        datasetColumn.fieldName = fieldName;
-        datasetColumn.fullFieldPath = fullPath;
-        datasetColumn.dataType = dataType;
-        datasetColumn.distributed = distributed;
-        datasetColumn.partitioned = partitioned;
-        datasetColumn.indexed = indexed;
-        datasetColumn.nullable = nullable;
-        datasetColumn.comment = comment;
-        datasetColumn.commentCount = commentCount;
-        datasetColumn.treeGridClass = "treegrid-" + Integer.toString(datasetColumn.sortID);
-        if (datasetColumn.parentSortID != 0) {
-            datasetColumn.treeGridClass += " treegrid-parent-" + Integer.toString(datasetColumn.parentSortID);
+        datasetColumn.setId(id);
```

Comment:
Have we considered trying out other data access frameworks than springJDBC. E.g. for hibernate, this row mapper will not be needed but a few annotations on the java fields. The framework will do the DB column to POJO mapping for you.